support-frontend/assets/helpers/subscriptions.js

```diff
@@ -271,6 +271,18 @@ const getNewsstandSaving = (subscriptionMonthlyCost: number, newsstandWeeklyCost
 const getNewsstandPrice = (productOption: PaperProductOptions) =>
   newsstandPrices[productOption];
 
+
+// Paper product
+
+const paperHasDeliveryEnabled = (): boolean => {
+  try {
+    return window.guardian.settings.switches.experiments.hidePaperHomeDelivery.state === 'Off';
```

Comment:
Might be good to change this switch to `paperHomeDeliveryEnabled` so that it matches how it is used in the app rather than being the opposite?

Reply:
you are quite right

Reply:
Also you can get this from `state.common.settings` rather than directly from the window object